prototypes/armor.lua

```diff
@@ -81,7 +80,7 @@ data:extend({
             type = "electric",
             usage_priority = "secondary-input"
         },
-        energy_consumption = "0W",
+        energy_consumption = "1W",
```

Comment:
this'll need testing to confirm it still works without power (maybe `"0"` might work?), since the armor itself has no slots for batteries